### PR TITLE
[REF] mail: `AttachmentViewer`, review and simplify scss

### PR DESCRIPTION
**Pull request by Brieuc-brd**
Part of the overall v16 SCSS optimization/restyle, task-2704984

task-2781992

--
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

### CODE REVIEW

**Comment by Brieuc-brd** [comment]
@stefanorigano 👋 
This PR is ready for review 👍

**Review by stefanorigano — CHANGES_REQUESTED** [review]

File: addons/mail/static/src/components/attachment_viewer/attachment_viewer.xml [hunk 1]
```diff
@@ -2,11 +2,11 @@
...
 
     <t t-name="mail.AttachmentViewer" owl="1">
-        <div class="o_AttachmentViewer" t-attf-class="{{ className }}" t-on-click="_onClick" t-on-keydown="_onKeydown" tabindex="0" t-ref="root">
+        <div class="o_AttachmentViewer flex-column align-items-center d-flex w-100 h-100" t-attf-class="{{ className }}" t-on-click="_onClick" t-on-keydown="_onKeydown" tabindex="0" t-ref="root">
             <t t-if="attachmentViewer">
-                <div class="o_AttachmentViewer_header" t-on-click="_onClickHeader">
+                <div class="o_AttachmentViewer_header d-flex w-100 bg-black-75 text-400" t-on-click="_onClickHeader">
                     <t t-if="attachmentViewer.attachment.isViewable">
-                        <div class="o_AttachmentViewer_headerItem o_AttachmentViewer_icon">
+                        <div class="o_AttachmentViewer_headerItem d-flex align-items-center ms-4 me-2 o_AttachmentViewer_icon">
```
Comment (stefanorigano) [inline addons/mail/static/src/components/attachment_viewer/attachment_viewer.xml:9]: Move utility-classes **after** odoo's custom classes

File: addons/mail/static/src/components/attachment_viewer/attachment_viewer.xml [hunk 2]
```diff
@@ -21,73 +21,73 @@
                             </t>
                         </div>
                     </t>
-                    <div class="o_AttachmentViewer_headerItem o_AttachmentViewer_name">
+                    <div class="o_AttachmentViewer_headerItem d-flex align-items-center mx-2 o_AttachmentViewer_name">
```
Comment (stefanorigano) [inline addons/mail/static/src/components/attachment_viewer/attachment_viewer.xml:24]: Again, move utility-classes **after** odoo's custom classes... could you check all the other lines?

**Review by stefanorigano — CHANGES_REQUESTED** [review]
@Brieuc-brd , just a couple of minor remarks.
When done you can remove the draft state and ping [@]seb for tech. review.

👍

File: addons/mail/static/src/components/attachment_viewer/attachment_viewer.scss [hunk 3]
```diff
@@ -3,134 +3,37 @@
...
-    transform: translateY(-50%);
 }
 
-.o_AttachmentViewer_buttonNavigationNext {
-    right: 15px;
-
-    > .fa {
-        margin: 1px 0 0 1px; // not correctly centered for some reasons
-    }
+.o_AttachmentViewer_buttonNavigationNext > .fa {
```
Comment (alexkuhn) [inline addons/mail/static/src/components/attachment_viewer/attachment_viewer.scss:14]: Don't deal with CSS specificity. Introduce classname `.o_AttachmentViewer_buttonNavigationNextIcon` on this element, and just target with this classname.

File: addons/mail/static/src/components/attachment_viewer/attachment_viewer.scss [hunk 4]
```diff
@@ -3,134 +3,37 @@
...
+    margin: 1px 0 0 1px; // not correctly centered for some reasons
 }
 
-.o_AttachmentViewer_buttonNavigationPrevious {
-    left: 15px;
-
-    > .fa {
-        margin: 1px 1px 0 0; // not correctly centered for some reasons
-    }
+.o_AttachmentViewer_buttonNavigationPrevious > .fa {
```
Comment (alexkuhn) [inline addons/mail/static/src/components/attachment_viewer/attachment_viewer.scss:18]: Same remark but with `.o_AttachmentViewer_buttonNavigationPreviousIcon`.

**Review by alexkuhn — COMMENTED** [review]
LGTM Otherwise

**Comment by alexkuhn** [comment]
(and resolve conflicts)

**Review by alexkuhn — APPROVED** [review]
@robodoo r+

**Comment by alexkuhn** [comment]
@robodoo rebase-ff

**Pull request closed without merging**